features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Listing/Listing.js

```diff
@@ -82,47 +82,51 @@ class Listing extends React.Component {
-                    <BottomNavigationButton label="Grid" icon={<GridOnIcon/>}/>
-                </BottomNavigation>
-                {
-                    this.state.apis ?
-                        this.state.listType === "grid" ?
-
-                            <Grid container>
-                                {this.state.apis.list.map(api => {
-                                    return <ApiThumb api={api} key={api.id}/>
-                                })}
-                            </Grid>
-                            :
-
-                            <Grid container>
-                                <Grid item xs>
-                                    <Paper>
-                                        <Table>
-                                            <TableHead>
-                                                <TableRow>
-                                                    <TableCell>Name</TableCell>
-                                                    <TableCell>Version</TableCell>
-                                                    <TableCell>Context</TableCell>
-                                                    <TableCell>Description</TableCell>
-                                                    <TableCell></TableCell>
-                                                </TableRow>
-                                            </TableHead>
-                                            <TableBody>
-                                                {this.state.apis.list.map(api => {
-                                                    return <APiTableRow api={api} key={api.id}/>
-                                                })}
-                                            </TableBody>
-                                        </Table>
-                                    </Paper>
-                                </Grid>
-                            </Grid>
-                        :
-                        <Loading/>
-                }
+            <div>
+                <Grid container justify="center" alignItems="center" className="full-width">
+                    <Grid item xs={12} sm={12} md={8} lg={8} xl={8}>
+                        <Typography type="display1" gutterBottom className="api-list-title">
+                            All APIs
+                        </Typography>
+                        <BottomNavigation value={value} onChange={this.handleChange} style={{float: "right"}}>
+                            <BottomNavigationButton icon={<ListIcon/>}/>
+                            <BottomNavigationButton icon={<GridOnIcon/>}/>
+                        </BottomNavigation>
+                        {
+                            this.state.apis ?
+                                this.state.listType === "grid" ?
+                                    <Grid container >
+                                        {this.state.apis.list.map(api => {
+                                            return <ApiThumb api={api} key={api.id}/>
```

Comment:
Same as below, if arrow function has only one statement no need to put curly braces and return, instead simply put the single statement inline